author: Andres Salomon <dilinger@debian.org>

Bookworm's older gn (generate-ninja) doesn't understand the "mnemonic"
variable. Since that string is used by tests that we delete, it doesn't
matter anyways.

Index: chromium-117.0.5938.62/third_party/blink/renderer/core/BUILD.gn
===================================================================
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -1677,7 +1677,6 @@ action_foreach("element_locator_test_pro
   python_path_root = "${root_out_dir}/pyproto"
   python_path_proto = "${python_path_root}/third_party/blink/renderer/core/lcp_critical_path_predictor"
 
-  mnemonic = "ELOC_PROTO"
 
   source_dir = "lcp_critical_path_predictor/test_proto"
   sources = rebase_path([ "lcp_image_id.asciipb" ], "", source_dir)
